sphinx/conf.py

```diff
@@ -23,11 +21,34 @@
 bf_version = '@bioformats.version@'
 
 import sys, os
-sys.path.insert(0, os.path.abspath(os.path.join(srcdir, '_ext')))
 import re
 import subprocess
 from datetime import datetime
 
+# Build with maven
+def getValue(value):
+    with open("../target/sphinx/conf.py") as f:
+        content = f.readlines()
+        for x in content:
+            if x.startswith(value):
+                values = x.split("=")
+                return values[1].strip().replace('\'', '')
+
+
+if os.environ.get('READTHEDOCS') == 'True':
+    os.system('mvn install -DskipSphinxTests=true -f ../pom.xml')
```

Comment:
Is there  a potential here for circular dependency since `mvn install` also wraps `sphinx-build`. Potentially only the relevant Maven phases should be invoked i.e. `generate-sources` ?

Reply:
I went for the "easiest" option at this stage of the work to see if it works, this needs to be broken up

Reply:
clarification: ``generate-sources`` will not be enough

Reply:
I will look into breaking it up. We also have ``zip`` step for example that we do not need